modules/mod_signal/mod_signal.erl

```diff
@@ -182,11 +195,11 @@ handle_call({'connect', Signal, Slot}, _From, State) ->
     TagSet = ets:lookup(State#state.slots, SignalType),
 
     Objects = case lists:member({tags, SignalType, SignalTags}, TagSet) of
-                  true ->
-                      {slot, Key, Slot};
-                  false ->
-                      [{slot, Key, Slot}, {tags, SignalType, SignalTags}]
-              end,
+        true ->
+            {slot, Key, Slot};
+        false ->
+            [{slot, Key, Slot}, {tags, SignalType, SignalTags}]
+    end,
```

Comment:
I can imagine. The problem with aligned below is that (depending on the preceding statement) they can jump all over the place and start very far to the right end leaving only a little line length.